src/main/java/uk/gov/hmcts/reform/sendletter/model/SendLetterResponse.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.sendletter.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.UUID;
+
+public class SendLetterResponse {
+
+    @ApiModelProperty(
+        name = "Letter Id",
+        notes = "Id of the letter sent to print"
+    )
+    @JsonProperty("letter_id")
+    public final UUID letterId;
+
+    public SendLetterResponse(UUID letterId) {
+        this.letterId = letterId;
+    }
+}
```

Comment:
i think we usually keep response objects under `domain` package. probably without `Response` suffix but not confident about this statement

Reply:
I am not sure why we need to have a `domain` package as normally when I would put any domain related entities/objects there and not DTO.Regarding naming convention please see JLS https://docs.oracle.com/javase/specs/jls/se9/html/jls-6.html#jls-6.

Reply:
ok no worries will pull out the response in domain packag 👍 .But filename can be `SendLetterResponse` and  not `SendLetter` ?Is that fine?